src/components/editable-title/index.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+import { Color } from '../colors';
+import { CopySize } from '../copy';
+import { getSpace, SpaceSize } from '../space';
+
+export enum EditState {
+	Editable = 'Editable',
+	Editing = 'Editing'
+}
+
+export interface EditableTitleProps {
+	fontSize?: CopySize;
```

Comment:
Could `fontSize` be bound to `secondary`?

Reply:
That totally makes sense. Removed the fontSize prop